hangman/game.py

```diff
@@ -1,25 +1,87 @@
 from .exceptions import *
+import random
 
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
-
-def _get_random_word(list_of_words):
-    pass
-
+""" Mask word"""
 
 def _mask_word(word):
-    pass
+  if word =='':
+    raise InvalidWordException
+    
+  else: 
+    mask_word = '*' * len(word) 
+    return mask_word
 
+""" Uncover word """
 
 def _uncover_word(answer_word, masked_word, character):
-    pass
-
+  answer_word = answer_word.lower()
+  character = character.lower()
+  list_ans = list(answer_word)
+  list_mask = list(masked_word)  
```

Comment:
`answer_word` and `character` are `string` data type ... `string` is actually also a collection like `list`.. so, you don't need to convert it to `list`... you may remove line 22 and 23